semeio/workflows/misfit_preprocessor/misfit_preprocessor.py

```diff
@@ -17,11 +18,16 @@
 
 class MisfitPreprocessorJob(SemeioScript):  # pylint: disable=too-few-public-methods
     def run(self, *args):
+        facade = LibresFacade(self.ert())
         config_record = _fetch_config_record(args)
-        measured_record = _load_measured_record(self.ert())
+        observations = _get_observations(facade)
+        config = assemble_config(config_record, observations)
+        config = config.snapshot
+
+        measured_record = _load_measured_record(facade, observations)
         scaling_configs = misfit_preprocessor.run(
             **{
-                "misfit_preprocessor_config": config_record,
+                "config": config,
                 "measured_data": measured_record,
                 "reporter": self.reporter,
             }
```

Comment:
Could I ask why we're not providing the arguments as normal? We're here pretty much just creating a dict just to unpack it again with `**`?

`scaling_configs = misfit_preprocessor.run(config=config,..` should do the same thing?

Reply:
They should give the same thing. I did not implement this originally, so can not say why it was done, would you like me to change it?

Reply:
I was a bit quick approving, so it's already done. Not a critical thing

Reply:
Sorry for being late to the dance @lars-petter-hauge and @oyvindeide. It was implemented as this due to my writing on records and future data flow in the workflows in other semeio issues at the same time. It was useful for the discussion, but should it leak into production code? Perhaps not!

Feel free to change it to more standard argument passing at any time if you feel like it 👍